README.md

```diff
@@ -88,7 +88,7 @@ See [CREATION](CREATION.md) for an explanation.
 ## Requirements
 
 * iOS 9.0+
-* Xcode 8.3+
+* Xcode 9.0+
```

Comment:
Maybe add somewhere in the README that, to use Xcode 8.3, the developer should use library version 0.1.5.

Reply:
For me it's something natural to look up for an older library version. Should be specified in the  release notes of the version.

Reply:
Can you add it to the changelog?